modules/openapi-generator/src/main/resources/python/api_doc_example.mustache

```diff
@@ -8,22 +8,26 @@ from pprint import pprint
 {{#hasAuthMethods}}
```

Comment:
Can you add a line here instantiating configuration?

Reply:
I believe configuration is instantiated in the `{{> python_doc_auth_partial}}` snippet